template.tpl

```diff
@@ -438,6 +471,10 @@ ___TEMPLATE_PARAMETERS___
             "event_name": "page_view",
```

Comment:
remove this if you rely on gtm.init, otherwise we will have duplicate page views when page_view is triggered (through a History change for example)